generator/templates/Default/src/components/menu/Menu.vue

```diff
@@ -10,9 +10,6 @@
       >
 
         <template #activator>
-          <v-list-item-action>
-            <v-icon>{{ item.icon }}</v-icon>
-          </v-list-item-action>
```

Comment:
Why remove this?

Reply:
Omdat die dubbel was, er zit een prepend icon in